src/components/MainPage/Importers/Importer.vue

```diff
@@ -46,8 +46,12 @@
       <div v-show="importing">
         {{ step }}
       </div>
-    </div>
-  </v-container>
+    </v-container>
+    <v-skeleton-loader
+      v-else
+      type="paragraph"
+    />
+  </div>
```

Comment:
it took less than a millisecond to decrypt one filed but ok let's add this

![image](https://user-images.githubusercontent.com/3001759/75480743-7a562600-59aa-11ea-8770-b57e53dbf3f8.png)